frontend/src/app/measure/measure-form/measure-form.component.ts

```diff
@@ -0,0 +1,126 @@
+import { Component, OnInit } from '@angular/core';
+import { map, Observable, switchMap } from 'rxjs';
+import {
+  KeyResultMeasure,
+  KeyResultService,
+} from '../../shared/services/key-result.service';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Location } from '@angular/common';
+import { getNumberOrNull } from '../../shared/common';
+import { Measure, MeasureService } from '../../shared/services/measure.service';
+
+@Component({
+  selector: 'app-measure-form',
+  templateUrl: './measure-form.component.html',
+  styleUrls: ['./measure-form.component.scss'],
+})
+export class MeasureFormComponent implements OnInit {
+  measure$!: Observable<Measure>;
+
+  measureForm = new FormGroup({
+    value: new FormControl<number>(0, [
+      Validators.required,
+      Validators.pattern('^[0-9]*$'),
+    ]),
+    measureDate: new FormControl<Date>(new Date(), [Validators.required]),
+    changeInfo: new FormControl<string>('', [Validators.required]),
```

Comment:
changeInfo and Initiatives are not required.

Reply:
Die initiatives und das measureDate sind nun nicht mehr required